rllib/tests/test_checkpoint_restore.py

```diff
@@ -117,13 +117,15 @@ def ckpt_restore_test(alg_name, tfe=False, object_store=False, replay_buffer=Fal
 
             # Compare buffer content with restored one.
             if replay_buffer:
-                data = alg1.local_replay_buffer.replay_buffers[
+                storage1 = alg1.local_replay_buffer.replay_buffers[
                     "default_policy"
-                ]._storage[42 : 42 + 42]
```

Comment:
Same as obove, we should still require that the storage is slicable to nicely create sharded buffers.
@sven1977 This is not blocking, I'd just like to see it. 
We can also add it later if it __getitem__ does not supporte slicing atm.

Reply:
@ArturNiederfahrenhorst The tests here apply slicing on a plain Python list to check the content of the buffer.

In contrast, creating a slice of a storage, which is a ring buffer, is not straight forward and depends on the operations required for a slice of a storage, e.g., for sharded buffers. Therefore, I did not implement slicing support on purpose.

However, I have just created a read-only storage view which is now returned when slicing a storage.